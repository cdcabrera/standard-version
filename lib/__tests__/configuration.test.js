const path = require('path');
const { getConfiguration } = require('../configuration');

describe('configuration', () => {
  it('should attempt to getConfiguration', () => {
    expect(getConfiguration(path.join(__dirname, './__mocks__/versionrc.json'))).toMatchSnapshot('getConfiguration');
  });
});
