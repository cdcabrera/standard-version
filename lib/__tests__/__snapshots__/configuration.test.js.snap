// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`configuration should attempt to getConfiguration: getConfiguration 1`] = `
Object {
  "types": Array [
    Object {
      "section": "Features",
      "type": "feat",
    },
    Object {
      "section": "Bug Fixes",
      "type": "fix",
    },
    Object {
      "hidden": true,
      "section": "Tests",
      "type": "test",
    },
    Object {
      "hidden": true,
      "section": "Build System",
      "type": "build",
    },
    Object {
      "hidden": true,
      "type": "ci",
    },
  ],
}
`;
